template/src/App.test.js

```diff
@@ -1,9 +1,8 @@
-import React from 'react';
-import { render } from '@testing-library/react';
+import { render, screen } from '@testing-library/react';
 import App from './App';
 
 test('renders learn react link', () => {
-  const { getByText } = render(<App />);
-  const linkElement = getByText(/learn react/i);
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
```

Comment:
This is done as per the code present in default [cra-template](https://github.com/facebook/create-react-app/blob/master/packages/cra-template/template/src/App.test.js)